flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -355,9 +424,12 @@
 	 * }
 	 * </pre>
 	 *
+	 * @deprecated Use {@link #sql(String)}.
+	 *
 	 * @param stmt The SQL statement to evaluate.
 	 * @param config The {@link QueryConfig} to use.
 	 */
+	@Deprecated
```

Comment:
Looks like this functionality can't be fully replaced with `sql()` because of the lack of `QueryConfig`